devex/vsextension/vsix-release-process.md

```diff
@@ -0,0 +1,78 @@
+# VS Extension Release Process
+
+The process used to build and release the nuget package from these
+sources is as follows.  Currently the signature on the release is
+a Microsoft signature generated using the Microsoft [PODSS](https://aka.ms/podss) process.
+In the future, this process will change to a use a non-Microsoft
+signature once another signing process is defined for Open Enclave.
+
+1. Build the latest [Cross-Platform Open Enclave SDK NuGet Package](https://github.com/HernanGatta/openenclave-cross-plat-nuget).
+   Make a note of the new version number for use in step 3.
+
+2. Make a note of the old version referenced in the VS extension sources.
+   This can be found by running the following in the vsextension directory:
+
+      ```cmd
+      findstr open-enclave-cross ProjectWizard\source.extension.vsixmanifest
```

Comment:
Sorry if this is obvious, Where do I run this command, is it in the open-enclave repo?

Reply:
yes.  after doing cd into devex/vsextension

Reply:
Updated text to clarify

Reply:
Thanks.